conf/services/penalties-appeals.yml

```diff
@@ -0,0 +1,27 @@
+serviceName: VAT Penalties and Appeals Reform
+serviceDescription: VAT-registered businesses, and their agents, can see penalties for late submission of VAT Returns and late payment of VAT, and appeal against those penalties online.
+serviceDomain: www.tax.service.gov.uk
+serviceUrl: /penalties-appeals.yml
```

Comment:
![image](https://user-images.githubusercontent.com/53828896/212308296-a0511d63-d23b-476f-b9f0-627ad135cae9.png)
This should be just `/penalties-appeals`